tests/Controller/ProductAttributeApiTest.php

```diff
@@ -85,6 +93,196 @@ public function it_allows_showing_product_attribute()
     }
 
     /**
+     * @test
+     */
+    public function it_does_not_allow_to_delete_product_attribute_if_it_does_not_exist()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+
+        $this->client->request('DELETE', '/api/v1/product-attributes/-1', [], [], static::$authorizedHeaderWithContentType);
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'error/not_found_response', Response::HTTP_NOT_FOUND);
+    }
+
+    /**
+     * @test
+     */
+    public function it_allows_delete_to_product_attribute()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $productAttributes = $this->loadFixturesFromFile('resources/product_attributes.yml');
+        $productAttribute = $productAttributes['productAttribute1'];
+
+        $this->client->request('DELETE', $this->getProductAttributeUrl($productAttribute), [], [], static::$authorizedHeaderWithContentType);
+
+        $response = $this->client->getResponse();
+        $this->assertResponseCode($response, Response::HTTP_NO_CONTENT);
+
+        $this->client->request('GET', '/api/v1/product-attributes/', [], [], static::$authorizedHeaderWithAccept);
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'product_attribute/index_response_after_delete', Response::HTTP_OK);
+    }
+
+    /**
+     * @test
+     */
+    public function it_allows_to_create_product_attribute()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $this->loadFixturesFromFile('resources/locales.yml');
+
+        $data = <<<EOT
+{
+    "code": "mug_material",
+    "type": "text",
+    "translations": {
+        "de_CH": {
+            "name": "Becher Material"
+        },
+        "en_US": {
+            "name": "Mug material"
+        }
+    }
+}
+EOT;
+
+        $this->client->request('POST', '/api/v1/product-attributes', [], [], static::$authorizedHeaderWithContentType, $data);
```

Comment:
I can change it - was mainly asking because it was explicitly requested to have the type in the payload when it was last added: https://github.com/Sylius/Sylius/pull/7398#issuecomment-276912669.